packages/shared/src/components/squads/SquadHeaderMenu.tsx

```diff
@@ -60,29 +61,37 @@ export default function SquadHeaderMenu({
   const items = useMemo(() => {
     const list: MenuItemProps[] = [
       {
-        Icon: TourIcon,
+        icon: <TourIcon size={IconSize.Small} className="mr-2" />,
         action: () =>
           openModal({
             type: LazyModal.SquadTour,
           }),
         label: 'Learn how Squads work',
       },
       {
-        Icon: FeedbackIcon,
+        icon: <FeedbackIcon size={IconSize.Small} className="mr-2" />,
         anchorProps: {
           href: `${squadFeedback}#user_id=${squad?.currentMember?.user?.id}&squad_id=${squad.id}`,
           target: '_blank',
         },
         label: 'Feedback',
       },
       canDeleteSquad
-        ? { Icon: TrashIcon, action: onDeleteSquad, label: 'Delete Squad' }
-        : { Icon: ExitIcon, action: onLeaveSquad, label: 'Leave Squad' },
+        ? {
+            icon: <TrashIcon size={IconSize.Small} className="mr-2" />,
+            action: onDeleteSquad,
+            label: 'Delete Squad',
+          }
+        : {
+            icon: <ExitIcon size={IconSize.Small} className="mr-2" />,
+            action: onLeaveSquad,
+            label: 'Leave Squad',
+          },
     ];
 
     if (canEditSquad) {
       list.unshift({
-        Icon: SettingsIcon,
+        icon: <SettingsIcon size={IconSize.Small} className="mr-2" />,
```

Comment:
Since this is quite repetitive, can we maybe abstract to inline icon or see if it's needed all togehter?
Think sidebar does something like that.

Reply:
Abstracted it to a component called `ContextMenuIcon` to work like the sidebar sharing the classes and size